ext/standard/string.c

```diff
@@ -1883,8 +1883,8 @@ PHP_FUNCTION(strpos)
 		offset += (zend_long)ZSTR_LEN(haystack);
 	}
 	if (offset < 0 || (size_t)offset > ZSTR_LEN(haystack)) {
-		php_error_docref(NULL, E_WARNING, "Offset not contained in string");
-		RETURN_FALSE;
+		zend_value_error("Offset not contained in string");
```

Comment:
IIRC those were denied when I did them in August

Reply:
Oh, thanks for the reminder! I'll remove this commit then.

Reply:
@Girgias I didn't find any trace about it among your PRs even though I looked at several of them. Could you please give me a hint?

Reply:
Maybe was misremembering in that case, @nikic does this ring a bell or did I just forget about this case?